splunk_add_on_ucc_framework/global_config.py

```diff
@@ -74,6 +80,41 @@ def __init__(self, global_config_path: str) -> None:
         self._original_path = global_config_path
         self.user_defined_handlers = UserDefinedRestHandlers()
 
+    def generate_minimal_globalconfig(
```

Comment:
I was thinking more about something like in this PR - #1655. We introduce the second way to initialize the `GlobalConfig` class which takes an instance of `AppConf` and an instance of `AppManifest` classes and should return `GlobalConfig` as a result. We do not need to dump it straight in this method, we have `dump` method in the `GlobalConfig` class which is designed to do it.

We might need another refactor PR to handle the `_original_path` parameter in the `GlobalConfig` class.

Reply:
Let's introduce this new `from_app_conf_and_app_manifest` method in a separate PR, you might need to make `_original_path` to be `Optional[str]` which may cause other parts of the class to be refactored.